### PR TITLE
Blogs Revamp

### PR DESCRIPTION
**Pull request by marcdhi**

### CODE REVIEW

**Comment by marcdhi** [comment]
@AahilRafiq Please start working on individual blog pages!

**Comment by AahilRafiq** [comment]
alright , working on it

File: src/app/loading.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export default function Component() {
```
Comment (marcdhi) [inline src/app/loading.js:1]: instead of this make a central component, which can be reused everywhere!

**Comment by AahilRafiq** [comment]
Changes Done

**Pull request merged**